README.md

```diff
@@ -332,6 +332,21 @@ else
 fi
 ```
 
+If you're using the SwiftLintPlugin from SPM. Use the
+following script implementation:
+
+```bash
+SWIFT_PACKAGE_DIR="${BUILD_DIR%Build/*}SourcePackages/artifacts"
+SWIFTLINT_CMD=$SWIFT_PACKAGE_DIR/swiftlintplugins/SwiftLintBinary/SwiftLintBinary.artifactbundle/swiftlint-*/bin/swiftlint
+
+if test -f $SWIFTLINT_CMD 2>&1
+then
+    $SWIFTLINT_CMD
```

Comment:
```suggestion
    "$SWIFTLINT_CMD"
```

Quotes around variables are a good practice in case of any spaces in file paths.

Reply:
I second what @SimplyDanny mentioned. Good to wrap it with spaces.

Reply:
Adding quote on script lines will break the script, give it a try and you will see.

Reply:
Yeah, then the second line needs to make sure the wildcard is properly resolved. I suggest
```bash
SWIFTLINT_CMD=$(ls "$SWIFT_PACKAGE_DIR"/swiftlintplugins/SwiftLintBinary/SwiftLintBinary.artifactbundle/swiftlint-*/bin/swiftlint | head -n 1)
```
to make sure potential spaces are quoted and there is only a single result in the command.